sum.js

```diff
@@ -0,0 +1,3 @@
+const sum = (a, b) => a + b;
+
+export default sum;
```

Comment:
- [ ] It would be better if you don't commit files not related to the project and added to .gitignore. Please add this `sum.js` file to `.gitignore` because this file doesn't make any functionality in the project. 💪 👀